src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/task/UpdateKpis.kt

```diff
@@ -28,13 +28,13 @@ class UpdateKpis(
     lockAtLeastFor = defaultLockAtLeastFor,
     lockAtMostFor = defaultLockAtMostFor,
   )
-  fun updateKpis() {
+  fun updateKpis(): Unit = runBlocking {
```

Comment:
This is a top-level non-suspend function so `runBlocking` can be lifted out to the top of it to reduce overhead of creating multiple blocking scopes